example/tools/lint/linters.bzl

```diff
@@ -52,9 +52,13 @@ pmd = lint_pmd_aspect(
 pmd_test = lint_test(aspect = pmd)
 
 checkstyle = lint_checkstyle_aspect(
-    binary = Label("@//tools/lint:checkstyle"),
-    config = Label("@//:checkstyle.xml"),
-    data = [Label("@//:checkstyle-suppressions.xml")],
+    binary = "@@//tools/lint:checkstyle",
+    config = "@@//:checkstyle.xml",
+    configs = {
+        "@@//:checkstyle.xml": "@@//src,@@//test",
```

Comment:
I'm taking another look at what apple_rules_lint did to permit users to configure linters per-directory, i.e. https://github.com/apple/apple_rules_lint/blob/main/api.md#get_lint_config allows you to set `tags` and https://github.com/apple/apple_rules_lint/blob/main/lint/private/package_lint_config.bzl#L14 allows the linter configuration to be selected in the same package being visited.

I'm curious, you must have considered just using apple_rules_lint instead of Aspect, since this checkstyle case is the example on their main README and I assume it motivated their design for selecting configs. Did you come to understand any tradeoffs between these approaches?